src/yglu/functions.py

```diff
@@ -29,8 +29,8 @@ def import_function(filename):
             import_dirs.extend(import_path.split(os.pathsep))
 
         allow = False
-        for dir in import_dirs:
-            relative_path = os.path.relpath(filepath, start=dir)
+        for my_dir in import_dirs:
```

Comment:
Could we give meaningful names like import_dir to renamed variables? my_ or a_ suffix are a bit noisy